src/main/java/com/commercetools/sync/inventories/InventoryService.java

```diff
@@ -0,0 +1,69 @@
+package com.commercetools.sync.inventories;
+
+import io.sphere.sdk.channels.Channel;
+import io.sphere.sdk.commands.UpdateAction;
+import io.sphere.sdk.inventory.InventoryEntry;
+import io.sphere.sdk.inventory.InventoryEntryDraft;
+
+import javax.annotation.Nonnull;
+import java.util.List;
+import java.util.Set;
+import java.util.concurrent.CompletionStage;
+
+/**
+ * Interface contains methods to perform sphere queries/commands related to Inventory topic.
+ * Listed operations are used in {@link InventorySync}.
+ */
+interface InventoryService {
+
+    /**
+     * Fetches all {@link InventoryEntry} objects whose sku is included in {@code skus}.
```

Comment:
a comment below is kind of obvious too. I would change entire comment with just:
Queries existing {@link InventoryEntry}'s against set of skus.